From d303ab580453b2d86945e7784a30fe68bd40d29a Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Fri, 23 Aug 2013 17:27:30 +0800
Subject: [PATCH 5/5] ENGR00275477 Recovey: Mess code when enter recovery mode

Init the text array to make sure default text is null

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 screen_ui.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/screen_ui.cpp b/screen_ui.cpp
index 8376341..f3306f3 100644
--- a/screen_ui.cpp
+++ b/screen_ui.cpp
@@ -86,6 +86,8 @@ ScreenRecoveryUI::ScreenRecoveryUI() :
     for (int i = 0; i < 5; i++)
         backgroundIcon[i] = NULL;
 
+    memset(text, 0, sizeof(text));
+
     pthread_mutex_init(&updateMutex, NULL);
     self = this;
 }
-- 
1.8.0

